specification/loadtestservice/LoadTestService.Management/models.tsp

```diff
@@ -0,0 +1,258 @@
+}
+
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "CheckQuotaAvailabilityResponse has a allOf relationship with Resource"
+@doc("Check quota availability response object.")
+model CheckQuotaAvailabilityResponse extends Resource {
+  @doc("Check quota availability response properties.")
+  @extension("x-ms-client-flatten", true)
+  properties?: CheckQuotaAvailabilityResponseProperties;
+}
+
+@doc("Check quota availability response properties.")
+model CheckQuotaAvailabilityResponseProperties {
+  @doc("True/False indicating whether the quota request be granted based on availability.")
+  isAvailable?: boolean;
+
+  @doc("Message indicating additional details to add to quota support request.")
+  availabilityStatus?: string;
+}
+
+@doc("Common properties for resources.")
+@armCommonDefinition("Resource", Azure.ResourceManager.CommonTypes.Versions.v3)
+@armCommonDefinition("Resource", Azure.ResourceManager.CommonTypes.Versions.v4)
+@armCommonDefinition("Resource", Azure.ResourceManager.CommonTypes.Versions.v5)
+model Resource {
+  @doc("Fully qualified resource ID for the resource. Ex - /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}")
+  @visibility("read")
+  id?: string;
+
+  @doc("The name of the resource.")
+  @visibility("read")
+  name?: string;
+
+  @doc("The type of the resource. E.g. \"Microsoft.Compute/virtualMachines\" or \"Microsoft.Storage/storageAccounts\"")
+  @visibility("read")
+  type?: string;
+
+  @doc("Azure Resource Manager metadata containing createdBy and modifiedBy information.")
+  @visibility("read")
+  systemData?: SystemData;
+}
+
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "QuotaBucketRequest has a allOf relationship with Resource"
+@doc("Request object of new quota for a quota bucket.")
+model QuotaBucketRequest extends Resource {
+  @extension("x-ms-client-flatten", true)
+  @doc("Request object of new quota for a quota bucket.")
+  properties?: QuotaBucketRequestProperties;
+}
+
+@doc("New quota request request properties.")
+model QuotaBucketRequestProperties {
+  @doc("Current quota usage of the quota bucket.")
+  @minValue(0)
+  currentUsage?: int32;
+
+  @doc("Current quota limit of the quota bucket.")
+  @minValue(0)
+  currentQuota?: int32;
+
+  @doc("New quota limit of the quota bucket.")
+  @minValue(0)
+  newQuota?: int32;
+
+  @doc("Dimensions for new quota request.")
+  dimensions?: QuotaBucketRequestPropertiesDimensions;
+}
+
+@doc("Dimensions for new quota request.")
+model QuotaBucketRequestPropertiesDimensions {
+  @doc("Subscription Id dimension for new quota request of the quota bucket.")
+  @minLength(1)
+  subscriptionId?: string;
+
+  @doc("Location dimension for new quota request of the quota bucket.")
+  @minLength(1)
+  location?: string;
+}
+
+@doc("Values returned by the List operation.")
+model OutboundEnvironmentEndpointCollection {
```

Comment:
Page<T> defines exactly this. Please reuse instead of self-define.  https://azure.github.io/typespec-azure/docs/reference/azure.core#models